Mathlib/Analysis/Normed/Module/Basic.lean

```diff
@@ -50,6 +50,9 @@ variable [NormedSpace 𝕜 E] [NormedSpace 𝕜 F]
 instance (priority := 100) NormedSpace.isBoundedSMul [NormedSpace 𝕜 E] : IsBoundedSMul 𝕜 E :=
   IsBoundedSMul.of_norm_smul_le NormedSpace.norm_smul_le
```

Comment:
This can be removed.

Reply:
Looks like it wasn't redundant after all.

Reply:
Ah, i guess it's needed for the following line but never again.